junebug/stores.py

```diff
@@ -131,3 +132,14 @@ def store_status(self, channel_id, status):
         component.'''
         key = self.get_key(channel_id)
         return self.store_property(key, status['component'], status.to_json())
+
+    @inlineCallbacks
+    def get_statuses_dict(self, channel_id):
+        '''Returns the latest status message for each component in a
+        dictionary'''
+        key = self.get_key(channel_id)
+        statuses = yield self.load_all(key)
+        res = {}
+        for k, v in statuses.iteritems():
+            res[k] = json.loads(v)
```

Comment:
#46 adds a [`api_from_status`](https://github.com/praekelt/junebug/blob/feature/issue-46-post-received-events-to-status-url-in-channel-status-worker/junebug/utils.py#L97-L105) utility. I think it would help for us to use that here for consistency and so that we aren't passing through undocumented message fields only relevant to vumi (like `helper_metadata`).